docs/framework/unmanaged-api/debugging/cordebugstatechange-enumeration.md

```diff
@@ -1,53 +1,50 @@
   - "CorDebugStateChange"
-api_location: 
+api_location:
   - "mscordbi.dll"
-api_type: 
+api_type:
   - "COM"
 ms.assetid: 1d4424ab-5143-4e50-a84a-ceeb4ddf3bba
-topic_type: 
+topic_type:
   - "apiref"
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # CorDebugStateChange Enumeration
-Describes the amount of cached data that must be discarded based on changes to the process.  
-  
-## Syntax  
-  
-```  
-typedef enum CorDebugStateChange  
-{  
-    PROCESS_RUNNING = 0x0000001,   
-    FLUSH_ALL       = 0x0000002,   
-} CorDebugStateChange;  
-```  
-  
-## Members  
-  
-|Member|Description|  
-|------------|-----------------|  
-|`PROCESS_RUNNING`|The process reached a new memory state via forward execution.|  
-|`SET_CONTEXT_FLAG_UNWIND_FRAME`|The process' memory may be arbitrarily different than it was previously.|  
-  
-## Remarks  
- A member of the `CorDebugStateChange` enumeration is provided as an argument when the debugger calls the [ProcessStateChanged](../../../../docs/framework/unmanaged-api/debugging/icordebugprocess6-processstatechanged-method.md) method  
-  
-> [!NOTE]
->  This enumeration is intended for use in .NET Native debugging scenarios only.  
-  
-## Requirements  
- **Platforms:** See [System Requirements](../../../../docs/framework/get-started/system-requirements.md).  
-  
- **Header:** CorDebug.idl, CorDebug.h  
-  
- **Library:** CorGuids.lib  
-  
- **.NET Framework Versions:** [!INCLUDE[net_46_native](../../../../includes/net-46-native-md.md)]  
-  
+Describes the amount of cached data that must be discarded based on changes to the process.
+
+## Syntax
+
+```
+typedef enum CorDebugStateChange
+{
+    PROCESS_RUNNING = 0x0000001,
+    FLUSH_ALL       = 0x0000002,
+} CorDebugStateChange;
+```
+
+## Members
+
+| Member            | Description                                                              |
+| ----------------- | ------------------------------------------------------------------------ |
+| `PROCESS_RUNNING` | The process reached a new memory state via forward execution.            |
+| `FLUSH_ALL`       | The process' memory may be arbitrarily different than it was previously. |
+
+## Remarks
+ A member of the `CorDebugStateChange` enumeration is provided as an argument when the debugger calls the `ProcessStateChanged` method either with [ICorDebugProcess4::ProcessStateChanged](../../../../docs/framework/unmanaged-api/debugging/icordebugprocess4-processstatechanged-method.md) or [ICorDebugProcess6::ProcessStateChanged](../../../../docs/framework/unmanaged-api/debugging/icordebugprocess6-processstatechanged-method.md)
+
+## Requirements
+ **Platforms:** See [System Requirements](../../../../docs/framework/get-started/system-requirements.md).
+
+ **Header:** CorDebug.idl, CorDebug.h
+
+ **Library:** CorGuids.lib
+
+ **.NET Framework Versions:** [!INCLUDE[net_46_native](../../../../includes/net-46-native-md.md)]  and [!INCLUDE[net_current_v45plus](../../../../includes/net-current-v20plus-md.md)]
```

Comment:
This doesn't seem right

![image](https://user-images.githubusercontent.com/12971179/52514472-ef4f9580-2bc6-11e9-827e-a30a95a94b94.png)